src/StatsMsg.chpl

```diff
@@ -343,7 +343,7 @@ module StatsMsg {
             var eOut = st.addEntry(rname, outShape, t);
 
             const DD = domOffAxis(eIn.a.domain, axis);
-            forall idx in DD {
```

Comment:
just want to make sure this bit is intentional since it seems unrelated to the compat modules

Reply:
I didn't highlight all of it apparently but the `forall` is being turned into a `for`

Reply:
yeah, that change fixed an error I was seeing for nd-array builds. Just happened to notice it while working on this.